README.rst

```diff
@@ -45,7 +46,19 @@ The quick way::
 For more details see the install section in the documentation:
 https://spidermon.readthedocs.io/en/latest/installation.html
 
+For implementing any of Spidermon's amazing features such as `actions <https://spidermon.readthedocs.io/en/latest/actions.html>`_ or `item validations <https://spidermon.readthedocs.io/en/latest/item-validation.html>`_. We recommend to check the documentation for instructions to download specific packages needed for the, to work.  
+
```

Comment:
Instead of adding this here, I would simply modify the `pip` command to follow the recommended installation instructions, and let users read the installation documentation if they are looking for something more.

Reply:
I know right, I even had the same views on the topic. But after reading #140 and #88, I thought mentioning here can nudge users in the right direction. As everyone expects packages to come with `Batteries included`

That's another reason why I am working #141, I was thinking of adding a note to every action on what to install before using them. Like Slack requires the SlackClient and so and so. More information the better.

Reply:
> I would simply modify the pip command to follow the recommended installation instructions.

By this I meant changing the suggested `pip` command to `pip install "spidermon[monitoring,validation]"`, which [is what the documentation recommends](https://spidermon.readthedocs.io/en/latest/installation.html).

Reply:
Got it. Will remove the warning as well. This solves everything.